core/osrd-reporting/src/main/java/fr/sncf/osrd/reporting/exceptions/ErrorType.java

```diff
@@ -104,6 +104,10 @@ public enum ErrorType {
     UnknownRoute("unknown_route", "unknown route", ErrorCause.USER),
     DuplicateRoute("duplicate_route", "Two routes have the same name", ErrorCause.USER),
     MissingAttributeError("missing_attribute", "referencing missing attribute", ErrorCause.INTERNAL),
+    MissingSignalOnRouteTransition(
```

Comment:
You may disregard this comment if a `stopAtBufferStop` necessarily implies a route as well as a block transition.

`MissingSignalOnBlockTransition`.

Reply:
Blocks are something the user can't really edit or control, they're generated from routes. From their point of view it's only a route error.